setup.py

```diff
@@ -16,7 +16,8 @@
 data_files = []
 
 
-install_reqs = ['appdirs', 'colorama>=0.3.3', 'jinja2', 'six', 'enum34']
+install_reqs = ['appdirs', 'colorama>=0.3.3', 'jinja2', 'six',
+    'enum34;python_version<"3.4"']
```

Comment:
I would indent this further, but no need to have that hold up fixing the install.

Reply:
True, well the linter (yes we have one :smile: ) didn't complain, but you're right I could have at least tried something like:
```
install_reqs = [
    'appdirs', 'colorama>=0.3.3', 'jinja2', 'six', 'enum34;python_version<"3.4"'
]
```